client/my-sites/checkout/composite-checkout/wpcom/types/wpcom-store-state.ts

```diff
@@ -745,6 +752,16 @@ export const managedContactDetailsUpdaters: ManagedContactDetailsUpdaters = {
 		};
 	},
 
+	updateAccountEmail: (
```

Comment:
Nice job navigating the complexities of this data object and its updaters.

Reply:
Thanks, although I basically just looked at the existing `updateVatId` (and other code for the VAT ID field) and copied exactly what it did.  I don't claim to actually understand the details of what I was doing :smile: